packages/editor-ui/src/components/ActivationModal.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+	<Modal
+		:name="WORKFLOW_ACTIVE_MODAL_KEY"
+		width="460px"
+		minWidth="350px"
+		:title="$locale.baseText('activationModal.workflowActivated')"
+	>
+		<template v-slot:content>
+			<div>
+				<n8n-text>{{triggerContent}}</n8n-text>
+			</div>
+			<div :class="$style.spaced">
+				<n8n-text>
+					<n8n-text :bold="true">
+						{{$locale.baseText('activationModal.theseExecutionsWillNotShowUp')}}
+					</n8n-text>
+					{{ $locale.baseText('activationModal.butYouCanSeeThem') }}
+					<a @click="showExecutionsList">
+						{{ $locale.baseText('activationModal.executionList') }}
+					</a>
+					{{ $locale.baseText('activationModal.ifYouChooseTo') }}
+					<a @click="showSettings">{{ $locale.baseText('activationModal.saveExecutions') }}</a>
+				</n8n-text>
+			</div>
+		</template>
+
+
+		<template v-slot:footer="{ close }">
+			<div :class="$style.footer">
+				<el-checkbox :value="checked" @change="handleCheckboxChange">{{ $locale.baseText('activationModal.dontShowAgain') }}</el-checkbox>
+				<n8n-button @click="close" :label="$locale.baseText('activationModal.gotIt')" />
+			</div>
+		</template>
+	</Modal>
+</template>
+
+<script lang="ts">
+
+import Vue from 'vue';
+
+import Modal from '@/components/Modal.vue';
+import { WORKFLOW_ACTIVE_MODAL_KEY, EXECUTIONS_MODAL_KEY, WORKFLOW_SETTINGS_MODAL_KEY, LOCAL_STORAGE_ACTIVATION_FLAG } from '../constants';
+import { getActivatableTriggerNodes, getTriggerNodeServiceName } from './helpers';
+
+export default Vue.extend({
+	name: 'ActivationModal',
+	components: {
+		Modal,
+	},
+	props: [
+		'modalName',
+	],
+	data () {
+		return {
+			WORKFLOW_ACTIVE_MODAL_KEY,
+			checked: false,
+		};
+	},
+	methods: {
+		async showExecutionsList () {
+			this.$store.dispatch('ui/openModal', EXECUTIONS_MODAL_KEY);
+		},
+		async showSettings() {
+			this.$store.dispatch('ui/openModal', WORKFLOW_SETTINGS_MODAL_KEY);
+		},
+		handleCheckboxChange (checkboxValue: boolean) {
+			this.checked = checkboxValue;
+			window.localStorage.setItem(LOCAL_STORAGE_ACTIVATION_FLAG, checkboxValue.toString());
+		},
+	},
+	computed: {
+		triggerContent (): string {
+			const foundTriggers = getActivatableTriggerNodes(this.$store.getters.workflowTriggerNodes);
+			if (!foundTriggers.length) {
+				return '';
+			}
+
+			if (foundTriggers.length > 1) {
+				return this.$locale.baseText('activationModal.yourTriggersWillNowFire');
+			}
+
+			const trigger = foundTriggers[0];
+
+			const triggerNodeType = this.$store.getters.nodeType(trigger.type);
+			if (triggerNodeType.activationMessage) {
+				return triggerNodeType.activationMessage;
+			}
+
+			const serviceName = getTriggerNodeServiceName(triggerNodeType.displayName);
+			if (trigger.webhookId) {
```

Comment:
Maybe it would be good if we know the type of "trigger" and defined above some interface, just to be sure what should contain the trigger object.

Reply:
type is inferred. do we need any more info here?
<img width="268" alt="Screenshot 2022-01-20 at 17 42 09" src="https://user-images.githubusercontent.com/4711238/150382763-1080f6cc-c2cf-42cd-b5c8-df0a08e40006.png">

Reply:
Gottcha :)